### ISSUE TITLE
rect errors from Highstock without navigator

### ISSUE THREAD

**Issue by gvethon** [highcharts/highcharts#7041]
#### Expected behaviour
Highstock shouldn't log errors then naivgator is disabled:
Error: <rect> attribute width: Expected length, "NaN".

#### Actual behaviour
Removing series from highstock chart cause errors in console. This error occurs only, then navigator is disabled and from version 5.0.7.

#### Live demo with steps to reproduce
Click on 'Clear series' log errors on console. Then adding and removing series logs same errors. If navigator is enabled these errors don't occur.
http://jsfiddle.net/9698hyxj/4/

**Comment by pawelfus** [comment]
Hi @gvethon 

Thanks for reporting this issue and all of the details. You are right, it's a regression after 1204a14bfe30e101a1930a68ba6218a504450d98 commit

**Workaround:**
Add `redraw` callback and set width for the navigator axis: http://jsfiddle.net/9698hyxj/6/
```
chart: {
  events: {
    redraw: function () {
      this.navigator.xAxis.width = this.xAxis[0].width;
    }
  }
},
```

**Comment by pawelfus** [comment]
Fixed since v6.0.3, demo: http://jsfiddle.net/9698hyxj/7/
